benchmarks/text_hybrid_scoreboard/treedb_adapter/main.go

```
@@ -0,0 +1,369 @@
+	must(err)
+	return append(encoded, '\n')
+}
+func mustRead(path string) []byte { value, err := os.ReadFile(path); must(err); return value }
+func must(err error) {
+	if err != nil {
+		fatalf("%v", err)
+	}
+}
+func fatalf(format string, args ...any) { fmt.Fprintf(os.Stderr, format+"\n", args...); os.Exit(1) }
+func cpuNanos(r syscall.Rusage) int64 {
+	return int64(r.Utime.Sec)*1e9 + int64(r.Utime.Usec)*1e3 + int64(r.Stime.Sec)*1e9 + int64(r.Stime.Usec)*1e3
```

golangci-lint (2.12.2)

[error] unnecessary conversion

(unconvert)